### ISSUE TITLE
New Dell Computer New Win10 New Firefox Latest DTA ... no downloads

### ISSUE THREAD

**Issue by georgejones314** [downthemall/downthemall#207]
example site: http://www.philfarrand.com/podcast/
older version of DTA works fine on my ancient N410c with Win/XP ... but this newer version ???

I get nothing added to the manager ... selecting for mp3's.
OS Name	Microsoft Windows 10 Home
Version	10.0.18362 Build 18362
Other OS Description 	Not Available
OS Manufacturer	Microsoft Corporation
System Name	DESKTOP-xxxxxx
System Manufacturer	Dell Inc.
System Model	Inspiron 3595
System Type	x64-based PC
System SKU	096C
Processor	AMD A9-9425 RADEON R5, 5 COMPUTE CORES 2C+3G, 3100 Mhz, 2 Core(s), 2 Logical Processor(s)
BIOS Version/Date	Dell Inc. 1.0.0, 6/19/2019
SMBIOS Version	3.1
Embedded Controller Version	255.255
BIOS Mode	UEFI
BaseBoard Manufacturer	Dell Inc.
BaseBoard Product	051KVJ
BaseBoard Version	A00
Platform Role	Mobile
Secure Boot State	On
PCR7 Configuration	Elevation Required to View
Windows Directory	C:\Windows
System Directory	C:\Windows\system32
Boot Device	\Device\HarddiskVolume1
Locale	United States
Hardware Abstraction Layer	Version = "10.0.18362.387"
User Name	DESKTOP-xxxx\xxxx
Time Zone	Eastern Standard Time
Installed Physical Memory (RAM)	4.00 GB
Total Physical Memory	3.87 GB
Available Physical Memory	949 MB
Total Virtual Memory	6.70 GB
Available Virtual Memory	1.22 GB
Page File Space	2.82 GB
Page File	C:\pagefile.sys
Kernel DMA Protection	Off
Virtualization-based security	Not enabled
Device Encryption Support	Elevation Required to View
Hyper-V - VM Monitor Mode Extensions	Yes
Hyper-V - Second Level Address Translation Extensions	Yes
Hyper-V - Virtualization Enabled in Firmware	Yes
Hyper-V - Data Execution Protection	Yes

**Comment by d7415** [comment]
It works fine here, on the "Links" tab, with the "Audio" filter ticked. Are the links selected in the selection window? (The initial window after clicking DownThemAll!)

**Comment by georgejones314** [comment]
yes ... the url in my bug report was just an example. it doesn't work for any site or download type (audio, etc.) ... I've used downloadthemall for years (decade?) on my xp machine without issues, so it's not user errorl

**Comment by d7415** [comment]
So to confirm, they *are* selected in the selection window, but just don't make it to the manager?

**Comment by nmaier** [comment]
Please give detailed steps of what you're doing? Like
1. go to example.org
2. click button
3. select this and that
4. hit this button
etc
